### ISSUE TITLE
Cannnot build 5.4.6 with latest packaging scripts

### ISSUE THREAD

**Issue by tobzsc** [OpenNebula/packages#9]
See log:

```
Executing(%install): /bin/sh -e /var/tmp/rpm-tmp.E3bTaW
+ umask 022
+ cd /root/rpmbuild/BUILD
+ '[' /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64 '!=' / ']'
+ rm -rf /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64
++ dirname /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64
+ mkdir -p /root/rpmbuild/BUILDROOT
+ mkdir /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64
+ cd opennebula-5.4.6
+ export DESTDIR=/root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64
+ DESTDIR=/root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64
+ ./install.sh
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/oneacct.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/oneshowback.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/oneacl.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onehost.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/oneimage.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/oneuser.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onevm.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onevnet.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onetemplate.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onegroup.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onedb.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onedatastore.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onecluster.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onezone.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onevcenter.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/oneflow.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/oneflow-template.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onesecgroup.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onevdc.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onevrouter.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onemarket.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onemarketapp.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/onevmgroup.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-allocate-address.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-associate-address.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-attach-volume.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-create-keypair.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-create-volume.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-delete-keypair.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-delete-volume.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-describe-addresses.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-describe-images.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-describe-instances.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-describe-keypairs.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-describe-volumes.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-detach-volume.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-disassociate-address.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-reboot-instances.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-register.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-release-address.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-run-instances.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-start-instances.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-stop-instances.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-terminate-instances.1.gz’: No such file or directory
cp: cannot stat ‘/root/rpmbuild/BUILD/opennebula-5.4.6/share/man/econe-upload.1.gz’: No such file or directory
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula.service /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/systemd/system/opennebula.service
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula-scheduler.service /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/systemd/system/opennebula-scheduler.service
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula-sunstone.service /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/systemd/system/opennebula-sunstone.service
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula-gate.service /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/systemd/system/opennebula-gate.service
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula-econe.service /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/systemd/system/opennebula-econe.service
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula-flow.service /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/systemd/system/opennebula-flow.service
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula-novnc.service /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/systemd/system/opennebula-novnc.service
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula.conf /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/tmpfiles.d/opennebula.conf
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula.conf /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/tmpfiles.d/opennebula-sunstone.conf
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula.conf /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/tmpfiles.d/opennebula-gate.conf
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula.conf /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/tmpfiles.d/opennebula-flow.conf
+ install -p -D -m 644 share/pkgs/CentOS7/opennebula-node.conf /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/lib/tmpfiles.d/opennebula-node.conf
+ install -p -D -m 644 /root/rpmbuild/SOURCES/50-org.libvirt.unix.manage-opennebula.pkla /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/etc/polkit-1/localauthority/50-local.d/50-org.libvirt.unix.manage-opennebula.pkla
+ /usr/bin/mkdir -p /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/etc/sudoers.d
+ install -p -D -m 440 share/pkgs/CentOS/opennebula.sudoers /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/etc/sudoers.d/opennebula
+ /usr/bin/mkdir -p /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/etc/logrotate.d
+ install -p -D -m 644 share/pkgs/logrotate/opennebula /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/etc/logrotate.d/opennebula
+ install -p -D -m 644 src/oca/java/jar/org.opennebula.client.jar /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/usr/share/java/org.opennebula.client.jar
+ install -p -D -m 644 share/etc/sysctl.d/bridge-nf-call.conf /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/etc/sysctl.d/bridge-nf-call.conf
+ install -p -D -m 644 share/install_gems/CentOS7/Gemfile.lock /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/usr/share/one/Gemfile.lock
+ /usr/lib/rpm/find-debuginfo.sh --strict-build-id -m --run-dwz --dwz-low-mem-die-limit 10000000 --dwz-max-die-limit 110000000 /root/rpmbuild/BUILD/opennebula-5.4.6
extracting debug info from /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/usr/bin/oned
extracting debug info from /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/usr/bin/mm_sched
extracting debug info from /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/usr/bin/tty_expect
extracting debug info from /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/usr/lib/one/mads/collectd
/usr/lib/rpm/sepdebugcrcfix: Updated 4 CRC32s, 0 CRC32s did match.
cpio: opennebula-5.4.6/expr_arith.cc: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/expr_arith.hh: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/expr_arith.y: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/expr_bool.cc: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/expr_bool.hh: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/expr_bool.y: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/expr_parser.c: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/expr_parser.l: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/template_parser.c: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/template_parser.l: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/template_syntax.cc: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/template_syntax.hh: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/template_syntax.y: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/vm_file_var_syntax.cc: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/vm_file_var_syntax.hh: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/vm_file_var_syntax.y: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/vm_var_parser.c: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/vm_var_parser.l: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/vm_var_syntax.cc: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/vm_var_syntax.hh: Cannot stat: No such file or directory
cpio: opennebula-5.4.6/vm_var_syntax.y: Cannot stat: No such file or directory
10225 blocks
+ /usr/lib/rpm/check-buildroot
+ /usr/lib/rpm/redhat/brp-compress
+ /usr/lib/rpm/redhat/brp-strip-static-archive /usr/bin/strip
+ /usr/lib/rpm/brp-python-bytecompile /usr/bin/python 1
+ /usr/lib/rpm/redhat/brp-python-hardlink
+ /usr/lib/rpm/redhat/brp-java-repack-jars
Processing files: opennebula-5.4.6-1.x86_64
Provides: config(opennebula) = 5.4.6-1 opennebula = 5.4.6-1 opennebula(x86-64) = 5.4.6-1
Requires(rpmlib): rpmlib(CompressedFileNames) <= 3.0.4-1 rpmlib(FileDigests) <= 4.6.0-1 rpmlib(PayloadFilesHavePrefix) <= 4.0-1
Requires: /usr/bin/env
Processing files: opennebula-server-5.4.6-1.x86_64
error: File not found by glob: /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/var/lib/one/remotes/etc/*
Executing(%doc): /bin/sh -e /var/tmp/rpm-tmp.cvIGTy
+ umask 022
+ cd /root/rpmbuild/BUILD
+ cd opennebula-5.4.6
+ DOCDIR=/root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/usr/share/doc/opennebula-server-5.4.6
+ export DOCDIR
+ /usr/bin/mkdir -p /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/usr/share/doc/opennebula-server-5.4.6
+ cp -pr LICENSE /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/usr/share/doc/opennebula-server-5.4.6
+ cp -pr NOTICE /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/usr/share/doc/opennebula-server-5.4.6
+ exit 0


RPM build errors:
    File not found by glob: /root/rpmbuild/BUILDROOT/opennebula-5.4.6-1.x86_64/var/lib/one/remotes/etc/*
```

Seems like some scripting problems. Also something with the manpages.

**Comment by vholer** [comment]
Hello,

sorry for problems with the packaging. The manual pages for few last releases were pre-generated by a script, which wasn't public. Now, when 5.4.6 is released, we have fixed both master + one-5.4 (for any **future** release) branches and placed a working script inside `share/man/build.sh` (use from here if missing in your particular checkout https://github.com/OpenNebula/one/blob/152189215e2c67406181cb30fe6397a413842562/share/man/build.sh).

The manual pages need to be generated when creating a release archive, please check https://github.com/OpenNebula/packages/#create-archive.

Thank you and sorry for the problems,
Vlastimil Holer

**Comment by tobzsc** [comment]
And what is with the other error regarding the build error?

File not found by glob: /root/rpmbuild/BUILDROOT/opennebul
a-5.4.6-1.x86_64/var/lib/one/remotes/etc/*
